### ISSUE TITLE
ClientAccessServerEnabled parameter

### ISSUE THREAD

**Issue by MaratMussabekov** [MicrosoftDocs/office-docs-powershell#3835]
it states "The ClientAccessServerEnabled parameter specifies whether to enable IRM for Outlook on the web (formerly known as Outlook Web App) and Exchange ActiveSync."
Wouldn't the setting of the parameter to $False affect the remote connections via Outlook Anywhere as well?
Thank you

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d3564095-4afa-a88c-ceb0-87239fe172ac
* Version Independent ID: e1cc3500-43c7-33cb-0b0a-332601c9ec9a
* Content: [Set-IRMConfiguration (exchange)](https://docs.microsoft.com/en-us/powershell/module/exchange/encryption-and-certificates/set-irmconfiguration?view=exchange-ps#feedback)
* Content Source: [exchange/exchange-ps/exchange/encryption-and-certificates/Set-IRMConfiguration.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/encryption-and-certificates/Set-IRMConfiguration.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by yogkumgit** [comment]
@officedocsbot assign @yogkumgit

**Comment by get-itips** [comment]
Hello @MaratMussabekov 
Thanks for your feedback.
As the article says, the parameter affects the enablement of the feature on Outlook on the Web (Outlook Web Access) and ActiveSync here is more info https://docs.microsoft.com/en-us/exchange/enable-or-disable-information-rights-management-on-client-access-servers-exchange-2013-help

**Comment by MaratMussabekov** [comment]
Yes, I saw this article, but from my experience, it disables irm for remote
users as well

Would it be possible to contact article author?

Thank you

**Comment by get-itips** [comment]
Hello @chrisda your call here, thank you

**Comment by get-itips** [comment]
added suggestions by @chrisda

**Comment by yogkumgit** [comment]
@chrisda Please merge this PR #3968
